resource-aggregate/service/config.go

```diff
@@ -44,7 +44,7 @@ func (c *APIsConfig) Validate() error {
 }
 
 type GRPCConfig struct {
-	OwnerCacheExpiration time.Duration `yaml:"ownerCacheExpiration" json:"ownerCacheExpiration" default:"10m"`
+	OwnerCacheExpiration time.Duration `yaml:"ownerCacheExpiration" json:"ownerCacheExpiration" default:"1m"`
```

Comment:
default is not used by yaml parser - so you can remove it

Reply:
I see, I'll get rid of the defaults then.

Reply:
I've removed it from fields with `yaml` and `json` annotations, but kept it if `envconfig` was present (for example: 	https://github.com/plgd-dev/cloud/blob/v2/resource-directory/service/config.go#L63)

Reply:
It is not used anymore in any https://github.com/plgd-dev/cloud/blob/v2/<SERIVICE>/service/config.go.

Only https://github.com/plgd-dev/cloud/blob/v2/pkg/config pkg can contains env var for loading config path.

Reply:
Removed from everything except c2c stuff, which can be removed only after the services are refactored.